rules/fusesoc.bzl

```diff
@@ -69,7 +68,7 @@ def _fusesoc_build_impl(ctx):
         "--setup",
         "--build",
     ])
-    args.add(out_dir.path, format = "--build-root=%s")
+    args.add("--build-root=%.")
```

Comment:
Isn't this going to change the path to all the output files? `dirname` is now used for the output groups, but not fusesoc's actual build-root.

Reply:
I've changed the implementation to leave the build directory alone and create symlinks into it from a separate `out` directory that the output groups can use.

Let me know what you think

Reply:
Oops, my symlinks are pointing to the wrong place. I'll fix this in the next push.

Reply:
You're right, I've worked around this and kept the path the same.